src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/BlobCleanerTaskTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.tasks.processors.ContainerCleaner;
+
+import static java.util.Arrays.asList;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class BlobCleanerTaskTest {
+    @Mock
+    private ContainerCleaner containerCleaner;
+
+    private BlobCleanerTask task;
+
+    void setUp(ServiceConfiguration serviceConfiguration) {
+        task = new BlobCleanerTask(containerCleaner, serviceConfiguration);
+    }
+
+    @DisplayName("Get all configured containers and process only 1 available container")
```

Comment:
Maybe we could name these tests sth like `should process all enabled containers` and `should not process any disabled containers`? Up to you.

Reply:
Ok

Reply:
or just remove this annotation